app/src/cli.ts

```diff
@@ -85,6 +85,7 @@ async function addCustody(
     maxPriceAgeSec: 60,
     oracleType: { [oracleType]: {} },
     oracleAccount: tokenOracle,
+    oracleAuthority: PublicKey.default, // By default, permissionless oracle price update is not allowed.
```

Comment:
Assign this to the desired oracle authority publickey for permissionless custom oracle price updates